packages/features/bookings/Booker/useSlotReservationId.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: It would be lost on refresh, so we need to persist it.
+// Though, we are persisting it in a cookie(`uid` cookie is set through reserveSlot call)
+// but that becomes a third party cookie in context of embed and thus isn't accessible inside embed
```

Comment:
I plan to implement a storage layer for embed that would persist data in top window(embedder page) on behalf of the iframe so that things stay consistent b/w embed and non-embed.